origins_workflow/src/Form/AuditSettingsForm.php

```diff
@@ -0,0 +1,224 @@
+    ];
+
+    $form['audit_button_hover_text'] = [
+      '#type' => 'textfield',
+      '#size' => 130,
+      '#title' => $this->t('Audit button hover text'),
+      '#description' => $this->t('Text to be displayed when the editor hovers their mouse over the audit button.'),
+      '#default_value' => $config->get('audit_button_hover_text'),
+    ];
+
+    $form['audit_confirmation_text'] = [
+      '#type' => 'textfield',
+      '#size' => 130,
+      '#title' => $this->t('Audit confirmation text'),
+      '#description' => $this->t('Ask the editor to confirm that they have audited the content.'),
+      '#default_value' => $config->get('audit_confirmation_text'),
+    ];
+
+    // Get a list of all content types.
+    $options = [];
+    $all_content_types = NodeType::loadMultiple();
+    foreach ($all_content_types as $machine_name => $content_type) {
+      if (!in_array($machine_name, ['mas_rss', 'webform'])) {
+        $options[$machine_name] = $content_type->label();
+      }
+    }
+
+    $form['audit_content_types'] = [
+      '#type' => 'checkboxes',
+      '#options' => $options,
+      '#title' => $this->t('Content types to be audited'),
+      '#default_value' => $config->get('audit_content_types'),
+    ];
+
+    return parent::buildForm($form, $form_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    parent::submitForm($form, $form_state);
+
+    // Check to see if any new content types have been selected.
+    $config = $this->config('origins_workflow.auditsettings');
+    $old_content_type_list = $config->get('audit_content_types');
+    $new_content_type_list = $form_state->getValue('audit_content_types');
+    // Find content types that have just been added.
+    foreach ($new_content_type_list as $this_type) {
+      if (!$this_type) {
+        continue;
+      }
+      if (!$old_content_type_list[$this_type]) {
+        $this->addAuditField($this_type);
+      }
+    }
+    // Find content types that have just been removed.
+    foreach ($old_content_type_list as $this_type) {
+      if (!$this_type) {
+        continue;
+      }
+      if (!$new_content_type_list[$this_type]) {
+        if (!$this->removeAuditField($this_type)) {
+          return;
+        }
+      }
+    }
+
+    $this->config('origins_workflow.auditsettings')
+      ->set('audit_button_text', $form_state->getValue('audit_button_text'))
+      ->set('audit_button_hover_text', $form_state->getValue('audit_button_hover_text'))
+      ->set('audit_confirmation_text', $form_state->getValue('audit_confirmation_text'))
+      ->set('audit_content_types', $form_state->getValue('audit_content_types'))
+      ->save();
+  }
+
+  /**
+   * Remove audit field from the content type.
+   */
+  private function removeAuditField($type) {
+    // Remove audit field from this content type.
+    $field = FieldConfig::loadByName('node', $type, 'field_next_audit_due');
```

Comment:
Should be able to access this from EntityConfig manager and inject that as a dependency.